Content.Server/Xenoarchaeology/XenoArtifacts/Triggers/Components/ArtifactHeatTriggerComponent.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Content.Server.Xenoarchaeology.XenoArtifacts.Triggers.Components;
+
+// TODO: This should probably be generalized for cold temperature too,
+// but right now there is no sane way to make a freezer.
+
+/// <summary>
+///     Triggers artifact if its in hot environment or
+///     has contacted with a hot object (lit welder, lighter, etc).
+/// </summary>
+[RegisterComponent]
+public sealed class ArtifactHeatTriggerComponent : Component
+{
+    /// <summary>
+    ///     Minimal surrounding gas temperature to trigger artifact.
+    ///     Around 100 degrees celsius by default.
+    ///     Doesn't affect hot items temperature.
+    /// </summary>
+    [DataField("activationTemperature")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public float ActivationTemperature = 573;
```

Comment:
this is 300 degrees celsius by default, not 100 as suggested by the comment?

Reply:
Yep, its typo